### PR TITLE
Issue #5489: Fixed indentation in config_filters.xml & config_headers…

### PR DESCRIPTION
**Pull request by varunu28**
….xml

Related to #5489 Change in config_headers.xml & minor change in config_filters.xml

### CODE REVIEW

File: src/xdocs/config_header.xml [hunk 1]
```diff
@@ -143,8 +143,8 @@ line 5: ////////////////////////////////////////////////////////////////////
         </p>
         <source>
 &lt;module name=&quot;Header&quot;&gt;
-    &lt;property name=&quot;header&quot;
-              value=&quot;// Copyright (C) 2004 MyCompany\n// All rights reserved&quot;/&gt;
+  &lt;property name=&quot;header&quot;
+            value=&quot;// Copyright (C) 2004 MyCompany\n// All rights reserved&quot;/&gt;
```
Comment (rnveach) [inline src/xdocs/config_header.xml]: shouldn't this be indented +2?

Comment (varunu28) [reply src/xdocs/config_header.xml]: @rnveach So should it come under just the ending of property?

Comment (rnveach) [reply src/xdocs/config_header.xml]: `&lt;property` is indented to position 2, so `value=&quot;` should be indented to position 4.

Comment (varunu28) [reply src/xdocs/config_header.xml]: Done

**Pull request merged**